### PR TITLE
Add event workshop Innovative Lernumgebung

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
~Untranslated `page_classes` was broken for this, so restore the old helper (which is fine and works).~

@lennartdefacto needs some content updates I think and all images must be added. You can find old workshop images in the workshop remove PR here: #1344

### CODE REVIEW

**Comment by lennartdefacto** [comment]
https://staging.defacto.nl/

@sn3p please review

File: source/localizable/evenementenagenda/_events.de.md.erb [hunk 1]
```diff
@@ -31,4 +31,63 @@
...
+      <p>
+        Wir laden Sie ganz herzlich ein zum Workshop „Innovative Lernumgebung für eine lückenlose Qualitätssicherung - wie Sie mit digitalen Lösungen Ihr Personal im Krankenhaus qualifizieren und motivieren“.
+      </p>
+
+      <%= link_to "Weitere Informationen & Anmelden", "/evenementenagenda/workshop-innovative-lernumgebung.html" %>
+    </div>
+
+  </li>
+
+  <!-- <li class="event">
```
Comment (sn3p) [inline source/localizable/evenementenagenda/_events.de.md.erb]: Can remove this?

Comment (RonaldDijkstra) [reply source/localizable/evenementenagenda/_events.de.md.erb]: Yes done

**Comment by RonaldDijkstra** [comment]
All good now.

News:
https://staging.defacto.nl/

Event index: 
https://staging.defacto.nl/veranstaltungskalender/

Event detail:
https://staging.defacto.nl/veranstaltungskalender/workshop-innovative-lernumgebung/

**Pull request merged**